tasmota/xsns_34_hx711.ino

```diff
@@ -441,6 +464,9 @@ void HxShow(bool json)
     }
     weight = (float)Hx.weight / 1000;                // kilograms
   }
+  if (Hx.absconv_a != 0 && Hx.absconv_b != 0) {
+    weight = (float)Hx.absconv_a / 1e9 * Hx.raw + (float)Hx.absconv_b / 1e6;
+  }
```

Comment:
Am I right in the conclusion that using the raw value here makes current calibration efforts useless? How do you define the absconv parameters? By trial and error?

Reply:
No you are correct. No offense, I didn't find the implemented calibration combined with taring the weight upon startup useful for my use case (and wonder in which use case it is). The strategy I follow is to take a few raw weights for known masses, then run a simple linear regression, e.g. here: https://mycurvefit.com
It's not as integrated but is a good deterministic process. Let me know if you believe we should have a discussion about a different path!

Reply:
My implemented use case is based on a commercial litchen scale  like the Ikea one.

After power on the user adds an empty dish, selects the reset button to zero the result and then is able to weigh anything he adds to the dish. Hence the zero button in the GUI.

There are ofcourse other uses which may use your suggested solution.

As your implementation doesn't interfere with mine I'll add it now so users can get some experience with it.

Reply:
Interesting element you brought up!

`USE_HX711_GUI` is not defined for `FIRMWARE_SENSORS`. As a result I did never see this "zero button in GUI" you refered to. Maybe this should be changed?

Reply:
@arendst ping in case you missed this.